main.py

```diff
@@ -0,0 +1,80 @@
+import sys
+
+from activation_functions import ActivationFunctions
+from data_manager import DataManager
+from data_visualizer import DataVisualizer
+from network import Network
+from network_enums import ProblemType
+
+LEARNING_RATE = 0.1
+EPOCH_MAX = 1
+INTIIAL_SEED = 1
+HIDDEN_LAYERS_COUNT = 2
+HIDDEN_LAYER_NEURONS_COUNT = 10
+
+
+def __main__() -> None:
+    # Input arguments - train_data_path, test_data_path, problem_type (c for classification, r for regression), visualize_epochs, number_of_layers, number_of_neurons_per_layer, use_bias, activation_function, error_function, initial_seed
+    # Example: python script.py "classification/data.simple.test.100.csv" "classification/data.simple.test.100.csv" c 1 2 30 1 sigmoid cross_entropy 1
+
+    # Raise exception if the number of arguments is not correct
+    if len(sys.argv) < 4:
+        raise Exception("Not enough arguments")
+
+    # Parse input arguments
+    train_data_path = sys.argv[1]
+    test_data_path = sys.argv[2]
+    problem_type = __parse_problem_type(sys.argv[3])
+    # Below are the arguments that should have default values
+    visualize_epochs = sys.argv[4] = bool(sys.argv[4]) or False
+    hidden_layers_count = int(sys.argv[5]) or HIDDEN_LAYERS_COUNT
+    hidden_layer_neurons_count = int(sys.argv[6]) or HIDDEN_LAYER_NEURONS_COUNT
```

Comment:
Maybe initialize `hidden_layer_neurons_count` with: `sqrt(n*m)` or `2*N+1` where N -> number of input and M -> number of output. It should probably give better values than arbitral default value :)

Reply:
sqrt(n*m) sounds good. Doner